testsuite/tests/prometheus/zync/test_annotations.py

```diff
@@ -21,7 +21,7 @@
 @pytest.fixture(params=['zync', 'zync-que'])
 def poda(request):
     """Return zync pod object."""
-    pod = openshift().get_pod(request.param).object()
+    pod = openshift().dc.get_pods(request.param).object()
```

Comment:
and what if zync will become deployment one day?

Reply:
Then this one line will change